Valentine Kopotchel/1/bot_server.rb

```diff
@@ -0,0 +1,25 @@
+require 'rubygems'
+require 'json'
+require 'redis'
+require 'telegram/bot'
+redis = Redis.new(url: ENV['REDIS_URL'],
+                  password: ENV['REDIS_PASSWORD']) # ENV was set in Heroku
+h = redis.get('db')
+h = JSON.parse h.gsub('=>', ':')
```

Comment:
this is a dirty hack
convert data to json before saving it to redis
If you want, look at #push, #lindex, #llen ,etjpds of redis.
https://www.rubydoc.info/github/redis/redis-rb/Redis